scripts/table_updates/tests/test_update_data_table.py

```diff
@@ -35,22 +37,94 @@ def mock_synapse(syn):
     syn.get.return_value = MagicMock(path="path/to/clinical_file.csv")
     return syn, release_files_table_synid
 
+
 @pytest.fixture
 def mock_release_version():
     return "v1.0"
 
+
 @pytest.fixture
 def column_mapping_table():
     return pd.DataFrame(
-        {"genie_element": ["ETHNICITY_DETAILED", "PRIMARY_RACE_DETAILED", "SEQ_YEAR", "SAMPLE_TYPE_DETAILED"], 
-        "prissmm_element": ["naaccr_ethnicity_code", "naaccr_race_code_primary", "cpt_seq_date", "cpt_sample_type"],
-        "prissmm_form": ["patient_characteristics", "patient_characteristics", "cancer_panel_test", "cancer_panel_test"]}
-        )
+        {
+            "genie_element": [
+                "ETHNICITY_DETAILED",
+                "PRIMARY_RACE_DETAILED",
+                "SEQ_YEAR",
+                "SAMPLE_TYPE_DETAILED",
+                "ETHNICITY",
+                "PRIMARY_RACE",
+                "SEQ_YEAR",
+                "SAMPLE_TYPE",
+                "ETHNICITY_DETAILED",
+                "PRIMARY_RACE_DETAILED",
+                "SEQ_YEAR",
+                "SAMPLE_TYPE_DETAILED",
+            ],
+            "prissmm_element": [
+                "naaccr_ethnicity_code",
+                "naaccr_race_code_primary",
+                "cpt_seq_date",
+                "cpt_sample_type",
+                "naaccr_ethnicity_code",
+                "naaccr_race_code_primary",
+                "cpt_seq_date",
+                "cpt_sample_type",
+                "naaccr_ethnicity_code",
+                "naaccr_race_code_primary",
+                "cpt_seq_date",
+                "cpt_sample_type",
+            ],
+            "prissmm_form": [
+                "patient_characteristics",
+                "patient_characteristics",
+                "cancer_panel_test",
+                "cancer_panel_test",
+                "patient_characteristics",
+                "patient_characteristics",
+                "cancer_panel_test",
+                "cancer_panel_test",
+                "patient_characteristics",
+                "patient_characteristics",
+                "cancer_panel_test",
+                "cancer_panel_test",
+            ],
+            "main_genie_release": [
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+                "v1.0",
+            ],
+            "cohort": [
+                "NSCLC",
+                "NSCLC",
+                "NSCLC",
+                "NSCLC",
+                "CRC",
+                "CRC",
+                "CRC",
+                "CRC",
+                "",
+                "",
+                "",
+                "",
+            ],
+        }
+    )
+
```

Comment:
Ideally, the main_genie_release version should vary to retrieve different subsets of main GENIE columns. I've simplified it here for clarity, but this doesn't affect our ability to test table subsetting. An empty string ("") in the cohort column is used to test the all_cohort case.

Reply:
It seems we leave it to allow some flexibilities. Originally, the script is for all_cohort.